Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Tabular/TransporterConfig.hs

```diff
@@ -36,18 +36,18 @@ mkPersist
   [defaultQQ|
     TransporterConfigT sql=transporter_config
       merchantId MerchantTId
-      pickupLocThreshold Meters Maybe
-      dropLocThreshold Meters Maybe
+      pickupLocThreshold Meters
+      dropLocThreshold Meters
       rideTravelledDistThresholdWhenPickupOrDestIsDiff Meters Maybe
```

Comment:
remove `rideTravelledDistThresholdWhenPickupOrDestIsDiff` and  `rideTravelledDistThresholdWhenPickupAndDestIsSame` if not used.